python/Ganga/new_tests/GPI/Bugs/Savannah18729.py

```diff
@@ -13,7 +13,9 @@ def Savannah18729(self):
 
         tmpdir = tempfile.mktemp()
         os.mkdir(tmpdir)
-        self.fname = os.path.join(tmpdir, ' test.C')
+        ## Is this a test of files with a leading ' '  in the name? - rcurrie
+        #self.fname = os.path.join(tmpdir, ' test.C')
+        self.fname = os.path.join(tmpdir, 'test.C')
```

Comment:
You're right; this was just a typo on my part. For the record, this is JIRA bug GANGA-343